src/messaging/discord.ts

```diff
@@ -66,13 +69,13 @@ export function sendDiscordMessage(link: Link, store: Store) {
         const promises = [];
         for (const webhook of webhooks) {
           const {id, token} = getIdAndToken(webhook);
-          const client = new Discord.WebhookClient({id, token});
+          const client = new Discord.WebhookClient({id, token}, clientOptions);
 
           promises.push(
             new Promise((resolve, reject) => {
               client
                 .send({
-                  content: notifyText.join(' '),
+                  content: notifyText.length ? notifyText.join(' ') : null,
```

Comment:
this change should also fix the bug reported in #2723: Couldn't send discord message: "Message content must be a non-empty string"

Reply:
I suppose that makes sense. Weird that there wouldn't be any `content` to begin with, but makes sense.